.eslintrc.js

```diff
@@ -41,11 +136,20 @@ module.exports = {
   'rules': Object.assign({},
     todo,
     {
+      "max-len": [
+        "error",
+        {
+          "code": 200,
```

Comment:
200 is required to support things like this - https://github.com/silverstripe/silverstripe-admin/blob/1/client/src/lib/dependency-injection/tests/ApolloGraphqlScaffoldingContainer-test.js#L164

Presumably there previously wasn't a line limit

Reply:
Arguably things like that shouldn't exist, and should be refactored into something we can actually read and interpret. Something for us to consider when revisiting these rules.